.github/workflows/build-multi-os.yml

```diff
@@ -61,7 +61,9 @@ jobs:
     - name: Set PACKAGE_VERSION
       run: |
         echo ${{ steps.vars.outputs.tag }} > src/main/resources/base/ayab/package_version
-        sed -i 's/PACKAGE_VERSION/${{steps.vars.outputs.tag}}/' src/build/settings/base.json
+        # remove suffix from semver tag, as fbs does not support them
+        version_without_suffix=$(echo ${{ steps.vars.outputs.tag }} | sed -E 's/^(v?[0-9]+\.[0-9]+\.[0-9]+).*$/\1/')
```

Comment:
@t0mpr1c3 the "Set PACKAGE_VERSION" step occurs in each OS job (4 times) - do you see a chance to have this in the caching job only  so we get rid of this redundancy? Same applies for "Set variables" step in each job

Reply:
We could, yes.

Reply:
I'll do it in a separate PR because it will clash with #554

Reply:
I think the `sed` code needs to be

     sed -E 's/^v?([0-9]+\.[0-9]+\.[0-9]+).*$/\1/'

(we should get rid of the initial `v`)

Reply:
But it'll be fixed in #554 anyway, so I'll resolve this now.